src/l-tile-layer-wms.js

```diff
@@ -47,21 +45,6 @@ class LTileLayerWMS extends LLayer {
 
     const standardOptions = parse(schema, this);
     const nonStandardOptionsElement = this.getAttribute("options");
```

Comment:
`nonStandardOptionsElement` feels like the wrong variable name, something like **text** or **attr** may be better suffixes